code/src/App.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react'
+import React, { useState } from "react";
+import QuestionAboutName from "./components/QuestionAboutName";
+import QuestionAboutPainting from "./components/QuestionAboutPainting";
+import QuestionAboutAuthor from "./components/QuestionAboutAuthor";
+import Summary from "./components/Summary";
 
 export const App = () => {
+  const [answers, setAnswers] = useState([]);
+  const [count, setCount] = useState(0);
+  const onFormSubmit = (value) => {
+    answers.push(value);
+    setCount((prev) => prev + 1);
+  };
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      {count === 0 && <QuestionAboutName onFormSubmit={onFormSubmit} />}
```

Comment:
_**onFormSubmit**_ I think is not the best name for this function because it sounds like a form event listener but it's a method that is used after the actual form has been submitted.